src/scripts/oauth/http-refresh-token.service.js

```diff
@@ -8,22 +8,15 @@ function HttpRefreshTokenService($rootScope, $q, $injector, OAuthService, OAuthT
 
   const service = this;
   let cache = null;
-  const limiters = [
-    /www\.bungie.net\/Platform\/User\/*/,
-    /www\.bungie.net\/Platform\/Destiny\/*/
-  ];
+  const matcher = /www\.bungie.net\/Platform\/(User|Destiny)\//;
```

Comment:
Don't need the wild card at the end? Is that what was "not right"? I like this regex vs the array, much cleaner! 👍